### PR TITLE
chore: the big package renaming

### PR DESCRIPTION
**Pull request by JamesHenry**
Consolidate everything under their new scoped names, yes this is unreviewable, but fortunately it is a one-off! :)

Better docs will follow after this before 1.0.0 tomorrow

### CODE REVIEW

File: packages/eslint-plugin-tslint/src/index.ts [hunk 1]
```diff
@@ -98,11 +98,11 @@ export const rules = {
 
       /**
        * The user needs to have configured "project" in their parserOptions
-       * for typescript-eslint-parser
+       * for @typescript-eslint/parser
        */
       if (!parserServices || !parserServices.program) {
         throw new Error(
-          `You must provide a value for the "parserOptions.project" property for typescript-eslint-parser`
+          `You must provide a value for the "parserOptions.project" property for @typescript-eslint/parser`
```
Comment (armano2) [inline packages/eslint-plugin-tslint/src/index.ts]: we should copy this message to 
packages/eslint-plugin/lib/util.js -> `getParserServices`

File: packages/eslint-plugin-tslint/tests/index.spec.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ ruleTester.run('tslint/config', rules.config, {
   valid: [
     {
       code: 'var foo = true;',
-      parser: 'typescript-eslint-parser',
+      parser: '@typescript-eslint/parser',
```
Comment (armano2) [inline packages/eslint-plugin-tslint/tests/index.spec.ts]: can you move parser and parserOptions to 
```ts
const ruleTester = new RuleTester();
```
```ts
const ruleTester = new RuleTester({
  parserOptions: {
    ecmaVersion: 6,
    sourceType: 'module',
    ecmaFeatures: {}
  },
  parser: '@typescript-eslint/parser',
  project: './tests/tsconfig.json'
});

File: packages/eslint-plugin/lib/util.js [hunk 3]
```diff
@@ -117,11 +117,11 @@ exports.getParserServices = context => {
     !context.parserServices.program ||
     !context.parserServices.esTreeNodeToTSNodeMap
   ) {
-    // TODO - the message will require revisiting once the typescript-eslint-parser/typescript-estree finalises
+    // TODO - the message will require revisiting once the @typescript-eslint/parser work finalises
     //        their work around exposing the parser. They may require that there be a project config field in
     //        the eslint config, in which case we should check and/or report that here appropriately.
     throw new Error(
-      'This rule requires you to use `eslint-plugin-typescript/parser`.'
+      'This rule requires you to use `@typescript-eslint/eslint-plugin/parser`.'
```
Comment (armano2) [inline packages/eslint-plugin/lib/util.js]:
```suggestion
      `You must provide a value for the "parserOptions.project" property for @typescript-eslint/parser`
```

**Comment by JamesHenry** [comment]
@armano2 Thanks I've noted down your points, adding them to my follow up PR - lots of tweaks to make yet anyway and I don't want anyone starting any WIP without the big reorg/renaming

**Pull request merged**